sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/CosmosClientBuilder.java

```diff
@@ -4,6 +4,7 @@
 
 import com.azure.data.cosmos.internal.Configs;
 import com.azure.data.cosmos.internal.Permission;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
If Azure Core is a dependency `ImplUtils` contains `isNullOrEmpty` which removes the need of this import.

Reply:
Cosmos SDK got merged in Azure Core few weeks back, still understanding Azure libraries, will improve this in future PRs.